sdk/src/test/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpanTest.java

```diff
@@ -253,6 +253,32 @@ public void toSpanData_WithInitialAttributes() {
     assertThat(spanData.getTotalAttributeCount()).isEqualTo(attributes.size() + 1);
   }
 
+  @Test
+  public void toSpanData_snapshot() {
```

Comment:
Can you please provide more descriptive name for the test? What does it actually test?

Reply:
```suggestion
  public void toSpanData_spanDataDoesNotChangeWhenModifyingSpan() {
```

would that be better?

Reply:
I also updated the PR description.

Reply:
Improved in 9ac9a16.